ecc/bls381/g1.go

```diff
@@ -26,6 +26,101 @@ type G1Affine struct {
 	X, Y fp.Element
 }
 
+// G1ZZZ parameterized jacobian coordinates (x=X/ZZ, y=Y/ZZZ, ZZ**3=ZZZ**2)
+type G1ZZZ struct {
```

Comment:
should we export this new struct? if we want to, then we need a better name :-) If it's internal to the multiExp, start with lower case later to not export it

Reply:
Not sure if we should export it, it's faster when one the input is affine, otherwise the standard Jacobian coordinates are fine.